src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.h

```diff
@@ -9,7 +9,9 @@ typedef void (*EnSnowwdActionFunc)(struct EnSnowwd*, PlayState*);
 
 typedef struct EnSnowwd {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x50];
+    /* 0x144 */ ColliderCylinder collider;            /* inferred */
+    /* 0x190 */ s16 unk190;                         /* inferred */
+    /* 0x192 */ char pad192[2];
     /* 0x194 */ EnSnowwdActionFunc actionFunc;
```

Comment:
```suggestion
    /* 0x144 */ ColliderCylinder collider;
    /* 0x190 */ s16 unk190;
    /* 0x194 */ EnSnowwdActionFunc actionFunc;
```
The comments are no longer needed. Also, the padding is automatically filled in due to alignment